ground/gcs/src/inc/widgetnames.h

```diff
@@ -1,6 +1,18 @@
 #ifndef WIDGETNAMES_H
 #define WIDGETNAMES_H
 
-
+#define CONFIGGADGETWIDGET_SIDEBAR						"configGadgetSideBar"
+#define CONFIGGADGETWIDGET_SIDEBAR_VBOXLAYOUT			"configGadgetSideBarVBox"
+#define CONFIGGADGETWIDGET_HWSETTINGS					"configGadgetHwSettings"
+#define CONFIGGADGETWIDGET_VEHICLETYPE					"configGadgetVehicleType"
+#define CONFIGGADGETWIDGET_INPUT						"configGadgetInput"
+#define CONFIGGADGETWIDGET_OUTPUT						"configGadgetOutput"
+#define CONFIGGADGETWIDGET_ATTITUDE						"configGadgetAttitude"
+#define CONFIGGADGETWIDGET_STABILIZATION				"configGadgetStabilization"
+#define CONFIGGADGETWIDGET_MODULES						"configGadgetModules"
+#define CONFIGGADGETWIDGET_AUTOTUNE						"configGadgetAutotune"
+#define CONFIGGADGETWIDGET_CAMERASTAB					"configGadgetCameraStab"
+#define CONFIGGADGETWIDGET_TXPID						"configGadgetTxPID"
+#define CONFIGGADGETWIDGET_OSD							"configGadgetCameraOSD"
 
```

Comment:
This header is a bad idea because:
a) #defines pollute namespaces, totally unnecessary in c++
b) config plugin stuff belongs in plugins/config, each plugin is basically standalone from the other parts of GCS (obv. there are dependencies between certain plugins exposed through their public APIs)
c) I don't see any actual win over just using the literals where you need them?
d) this will cause a ton of merge conflicts if you intended it to be for all plugins (which is not the way GCS is designed as per b)
e) breaks the style guide https://github.com/d-ronin/dRonin/wiki/Development-Coding-standards#gcs-code-c (see http://wiki.qt.io/Coding_Conventions#Aesthetics)